git-extra/vimrc

```diff
@@ -19,17 +19,16 @@ set statusline=%<%F%h%m%r\ [%{&ff}]\ (%{strftime(\"%H:%M\ %d/%m/%Y\",getftime(ex
 "------------------------------------------------------------------------------
 " Only do this part when compiled with support for autocommands.
 if has("autocmd")
-    "Set UTF-8 as the default encoding for commit messages
+    " Set UTF-8 as the default encoding for commit messages
     autocmd BufReadPre COMMIT_EDITMSG,MERGE_MSG,git-rebase-todo setlocal fileencodings=utf-8
 
-    "Remember the positions in files with some git-specific exceptions"
+    " Remember the positions in files with some git-specific exceptions"
     autocmd BufReadPost *
       \ if line("'\"") > 0 && line("'\"") <= line("$")
-      \           && expand("%") !~ "COMMIT_EDITMSG"
+      \           && &ft !~# 'commit\|gitrebase'
       \           && expand("%") !~ "MERGE_EDITMSG"
       \           && expand("%") !~ "ADD_EDIT.patch"
-      \           && expand("%") !~ "addp-hunk-edit.diff"
-      \           && expand("%") !~ "git-rebase-todo" |
+      \           && expand("%") !~ "addp-hunk-edit.diff" |
```

Comment:
For somebody versed in vim's command language, this might all look very clear. To everybody else, this is just magic.

That includes me, and I need to understand those changes before I can merge the PR. Please take a little bit of time to explain in the commit message what you did there, at least enough for somebody e.g. stuck in an airplane without internet access (and therefore unable to look up documentation and things) to  be able to change the `&ft` line to their needs.